student/src/App.js

```diff
@@ -15,7 +16,8 @@ class App extends Component {
     <Route exact path='/'/>
     <Route path='/Login' component={Login}/>
     <Route exact path='/signup' component = {Signup}/>
-      <Route path="/quiz/" render={(props) => <Quiz {...props} />} />
+    <Route exact path='/start' component = {Start}/>
+    <Route path="/quiz/" render={(props) => <Quiz {...props} quizId={1} /> }  />
     </div>
```

Comment:
No need to keep the props `quizID={1}`, it was just for testing purpose

Reply:
done :+1: